definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-alt-service-process-server-nonprod.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
```

Comment:
I thought we were going to use:

```
*-alternative-service-nonprod.json
```

Has that changed or I misunderstood it?

Reply:
No change, i hadn't seen the file name that felix used, but i assumed it would be the same as the epic name.
Ill update it now

Reply:
Pardon me, this is for a different epic. I believe im correct @yarpo.
What do you think @madjava ?